src/Query.php

```diff
@@ -138,8 +138,9 @@ public function verify()
      */
     public function sanitize()
     {
-        // This was commented out because array filter will also remove 0, and we do not want this
-        // $this->aggregations = array_filter($this->aggregations);
+        $this->aggregations = array_filter($this->aggregations, static function ($value) {
+            return $value !== null && $value !== '';
+        });
```

Comment:
Here you can use the arrow function feature.

```suggestion
        $this->aggregations = array_filter(
            $this->aggregations,
            static fn($value) => $value !== null && $value !== ''
        );
```
I did not test if this will fit in one line.